### DISCUSSION TITLE
ModuleNotFoundError: No module named 'orc8r.protos'

### DISCUSSION THREAD

**Discussion by vivekshrivastava1516** [magma/magma#15341] [Q&A]
During Vagrant Magma Run We are getting below error to start magma service for the AGW.

Nov 23 11:24:51 magma-dev magmad[95978]:   import snowflake
Nov 23 11:24:51 magma-dev magmad[95978]: Traceback (most recent call last):
Nov 23 11:24:51 magma-dev magmad[95978]:   File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
Nov 23 11:24:51 magma-dev magmad[95978]:     return _run_code(code, main_globals, None,
Nov 23 11:24:51 magma-dev magmad[95978]:   File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
Nov 23 11:24:51 magma-dev magmad[95978]:     exec(code, run_globals)
Nov 23 11:24:51 magma-dev magmad[95978]:   File "/home/vagrant/magma/orc8r/gateway/python/magma/magmad/main.py", line 18, in <module>
Nov 23 11:24:51 magma-dev magmad[95978]:     from magma.common.grpc_client_manager import GRPCClientManager
Nov 23 11:24:51 magma-dev magmad[95978]:   File "/home/vagrant/magma/orc8r/gateway/python/magma/common/grpc_client_manager.py", line 19, in <module>
Nov 23 11:24:51 magma-dev magmad[95978]:     from magma.common.service_registry import ServiceRegistry
Nov 23 11:24:51 magma-dev magmad[95978]:   File "/home/vagrant/magma/orc8r/gateway/python/magma/common/service_registry.py", line 19, in <module>
Nov 23 11:24:51 magma-dev magmad[95978]:     from magma.configuration.exceptions import LoadConfigError
Nov 23 11:24:51 magma-dev magmad[95978]:   File "/home/vagrant/magma/orc8r/gateway/python/magma/configuration/__init__.py", line 12, in <module>
Nov 23 11:24:51 magma-dev magmad[95978]:     importlib.import_module(mod)
Nov 23 11:24:51 magma-dev magmad[95978]:   File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
Nov 23 11:24:51 magma-dev magmad[95978]:     return _bootstrap._gcd_import(name[level:], package, level)
Nov 23 11:24:51 magma-dev magmad[95978]: ModuleNotFoundError: No module named 'orc8r.protos'
Nov 23 11:24:51 magma-dev systemd[1]: magma@magmad.service: Main process exited, code=exited, status=1/FAILURE
Nov 23 11:24:51 magma-dev magmad[95988]: Traceback (most recent call last):
Nov 23 11:24:51 magma-dev magmad[95988]:   File "/usr/local/bin/service_util.py", line 20, in <module>
Nov 23 11:24:51 magma-dev magmad[95988]:     from magma.common.health.service_state_wrapper import ServiceStateWrapper
Nov 23 11:24:51 magma-dev magmad[95988]:   File "/home/vagrant/magma/orc8r/gateway/python/magma/common/health/service_state_wrapper.py", line 16, in <module>
Nov 23 11:24:51 magma-dev magmad[95988]:     from magma.common.redis.client import get_default_client
Nov 23 11:24:51 magma-dev magmad[95988]:   File "/home/vagrant/magma/orc8r/gateway/python/magma/common/redis/client.py", line 14, in <module>
Nov 23 11:24:51 magma-dev magmad[95988]:     from magma.configuration.service_configs import get_service_config_value
Nov 23 11:24:51 magma-dev magmad[95988]:   File "/home/vagrant/magma/orc8r/gateway/python/magma/configuration/__init__.py", line 12, in <module>
Nov 23 11:24:51 magma-dev magmad[95988]:     importlib.import_module(mod)
Nov 23 11:24:51 magma-dev magmad[95988]:   File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
Nov 23 11:24:51 magma-dev magmad[95988]:     return _bootstrap._gcd_import(name[level:], package, level)
Nov 23 11:24:51 magma-dev magmad[95988]: ModuleNotFoundError: No module named 'orc8r.protos'
Nov 23 11:24:51 magma-dev systemd[1]: magma@magmad.service: Control process exited, code=exited, status=1/FAILURE
Nov 23 11:24:51 magma-dev systemd[1]: magma@magmad.service: Failed with result 'exit-code'.
has context menu

**Comment by aniket021997** [comment]
Hi @vivekshrivastava1516 ,
We just tried to build agw from vagrant vm, magma services are up. Please find details and log snap below.
branch: master
agw built using :
	magma/bazel//scripts/build_and_run_bazelified_agw.sh
mme, sessiond and pipelined are running.

![image](https://github.com/magma/magma/assets/95931489/81a193d2-7731-4af6-8057-5ae47b561609)

![image](https://github.com/magma/magma/assets/95931489/be62158e-f886-43db-a220-a23c0297b652)
